light-client/src/evidence.rs

```diff
@@ -35,20 +33,19 @@ mod prod {
     #[derive(Clone, Debug)]
     pub struct ProdEvidenceReporter {
         peer_map: HashMap<PeerId, tendermint_rpc::Url>,
-        timeout: Option<Duration>,
+        timeout: Duration,
     }
 
-    #[contract_trait]
+    #[async_trait::async_trait]
```

Comment:
Can we import `async_trait::async_trait` and write this instead? It's a bit nicer on the eyes imho.

```suggestion
    #[async_trait]
```